resources/pushHandler.js

```diff
@@ -327,9 +325,7 @@ const uploadToContainer = (z, bundle, result) => {
           }
 
           //This is a backup error checker for the size of the file.
-          if (fileContents[0].size >= 100 * 1024 * 1024 || getStringByteSize(fileContents[0].content) >= 100 * 1024 * 1024) {
-            throw new Error(messages.BIG_FILE);
-          }
+          fileSizeChecker(fileContents[1].size || getStringByteSize(fileContents[1].content));
```

Comment:
Did you mean to change `fileContents[0]` for `fileContents[1]` ? 
Are you sure `fileContents[1]` exists? 
Could it be here where I get this error for `.size` on an undefined object?

Reply:
Yeah that seems to be where it is coming from, not sure why I swapped this for 1 at some point. This should fix your errors on Zapier as well.